src/background.js

```diff
@@ -23,6 +22,8 @@ const store = new Store()
 const log = require('electron-log')
 const logger = log.scope('ElectronMain')
 
+const search = require('./search.js').default
```

Comment:
Search logic moved to a JS file that's used by either background.js (electron) or Search.vue (webapp). It doesn't have any special libs in it.